scavenge/tutorial/03-scaffolding.md

```diff
@@ -0,0 +1,39 @@
+---
+order: 3
+---
+
+# Scaffolding
+
+We'll be using a CLI tool called [Starport](https://github.com/tendermint/starport) to create a new Cosmos SDK blockchain. To use `starport` [install it](https://docs.starport.network/intro/install.html) on your local machine:
+
+```
+curl https://get.starport.network/starport@v0.17.0! | bash
+```
+
+Scaffold anew Cosmos SDK blockchain using the `starport scaffold chain` command. The command accepts one argument: the Go module path that will be used for the project.
+
+By default a chain is scaffolded with a new empty Cosmos SDK module. Use the `--no-module` flag to skip module scaffolding.
+
+```
+starport scaffold chain github.com/cosmonaut/scavenge --no-module
+```
+
+This command created a new directory `scavenge` with a brand new Cosmos SDK blockchain. This blockchain doesn't have any application-specific logic yet, but it imports standard Cosmos SDK modules, such as `auth`, `bank`, `mint` and others.
+
+Change the current directory to `scavenge`:
+
+```
+cd scavenge
+```
+
+Inside the project directory you can execute other Starport commands to start a blockchain node, scaffold modules, messages, types, generate code, and much more.
+
+In a Cosmos SDK blockchain application-specific logic is implemented in separate modules. This keeps code easy to understand and reuse.
```

Comment:
```suggestion
In a Cosmos SDK blockchain, application-specific logic is implemented in separate modules. Using modules keeps code easy to understand and reuse.
```

Reply:
the ever-present "unclear reference" is one of our favorite grammar challenges, wdyt about "Using modules" to describe what "this" is?